src/models/Card/Card.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Again, why are we disabling linting? To overrule weak-typing (`any`)? We should be strong-typing our variables.